src/ol/source/UTFGrid.js

```diff
@@ -263,6 +263,7 @@ export class CustomTile extends Tile {
  * If not provided, `url` must be configured.
  * @property {string} [url] TileJSON endpoint that provides the configuration for this source.
  * Request will be made through JSONP. If not provided, `tileJSON` must be configured.
+ * @property {boolean} [wrapX=true] Whether to wrap the world horizontally.
```

Comment:
IMO horizontally is an ambiguous term when a projected plane is in play.

```suggestion
 * @property {boolean} [wrapX=true] Whether to wrap the map along the X-axis (equator) of the projection.
```

Reply:
@mprins The equator is definitely wrong in this context.

Reply:
indeed!

Reply:
I think that is a separate issue which affects all tile sources (except for DataTile and GeoTIFF where the option is used inconsistently).  Maybe the description used for vector tiles is better and should be used everywhere?